LICENSE

```diff
@@ -1,21 +0,0 @@
-MIT License
```

Comment:
please don't delete the license-file (but you could update the year)

Reply:
made changes